app/src/main/java/com/example/playlistmaker/data/MediaPlayerManager.kt

```diff
@@ -0,0 +1,67 @@
+package com.example.playlistmaker.data
+
+import android.media.AudioAttributes
+import android.media.MediaPlayer
+import com.example.playlistmaker.domain.MediaPlayerUseCase
+
+class MediaPlayerManager : MediaPlayerUseCase {
```

Comment:
🍏 Снова - это не UseCase, это можно назвать с натяжкой репозиторием, работающим со специфическими данными. Но удобнее сделать так:
```suggestion
class MediaPlayerManagerImpl: MediaPlayerManager {
```